### ISSUE TITLE
OOCSS and BEM

### ISSUE THREAD

**Issue by mikoSTAR** [briisk/briisk-css#2]
"We encourage some combination of OOCSS and BEM for these reasons:

It helps create clear, strict relationships between CSS and HTML
It helps us create reusable, composable components
It allows for less nesting and lower specificity
It helps in building scalable stylesheets
...
"

Taking into consideration shadow DOM of Angular2 there are no huge advantages of using BEM or BEM with OOCSS as it is describe.
Components are small and thus that we don't need to name classes inside using that methodology.

**Comment by dakolech** [comment]
Remember that Angular 2 is not the only framework we are using

**Comment by augkamil** [comment]
yes, we should split this rule into two - one where we can use shadow DOM, the other where we're not able to use shadow DOM. @mikoSTAR can you prepare it like this?
